Neuro/Minigames/Completion/Solvers/FuelEnginesSolver.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+using UnityEngine;
+
+namespace Neuro.Minigames.Completion.Solvers;
+
+[MinigameSolver(typeof(RefuelStage))]
+public sealed class FuelEnginesSolver : MinigameSolver<RefuelStage>
+{
+    public override IEnumerator CompleteMinigame(RefuelStage minigame, NormalPlayerTask task)
+    {
+        // TODO: Figure out if the button is stored somewhere so we dont have to do this
+        Vector3 position = Vector3.Lerp(minigame.greenLight.transform.position, minigame.redLight.transform.position, 0.5f) - new Vector3(0f, 0.5f, 0f);
+        yield return InGameCursor.Instance.CoMoveTo(position);
+        minigame.Refuel();
+        while (!minigame.complete)
+            yield return new WaitForFixedUpdate();
+        minigame.Close();
```

Comment:
Why do you call minigame.Close from here? Doesn't it get called automatically from the minigame itself?

Reply:
```suggestion
        yield return Sleep(0.2f);
        minigame.Close();
```

```suggestion
        yield return minigame.CoStartClose(0.2f);
```